### DISCUSSION TITLE
3 approving reviews?

### DISCUSSION THREAD

**Discussion by Be-ing** [tenacityteam/tenacity-legacy#157] [General]
Pull requests now show that 3 approving reviews are required. This seems excessive IMO and I anticipate the project would move very slowly if that continues.

**Comment by Semisol** [comment]
2? 1?

**Reply by Semisol** [reply]
I felt it was a bit too much, but I don't know what exactly other than 2.

**Reply by Be-ing** [reply]
2 seems reasonable to me. If it turns out to be too slow, it can be lowered to 1.

**Reply by Semisol** [reply]
Okay, done.

**Comment by Flyingmana** [comment]
On one of the projects I maintain 2 Reviews are working out good.
Requiring more then one did in several cases help finding issues with changes earlier, as different people look for different things on Review.

We did extend the rules a bit, to also count in Reviews of non-maintainers to a certain degree and allowing the maintainers to merge them on own assessment then.
This also helped to get more people involved into Reviewing to take some work off from the maintainers, so they can focus on PRs, which already did get some Reviews.

With the current state here, I would say keeping 3 may be better for a while, at least till things settle a bit.

**Reply by Be-ing** [reply]
Thank you for sharing your experience. What project are you referring to? I proposed requiring 2 reviewers in a project I maintain for the same reasons but it was not received very well because others thought it would go too slowly. I really like how it provides an on-ramp for people to become involved and hopefully become maintainers eventually.

**Reply by Flyingmana** [reply]
the Projekt is OpenMage https://github.com/OpenMage/magento-lts a fork of Magento.
It may be a bit different for other projects. For OpenMage the primary reason for the Reviews is stability and additional security. (If you are trying to be PCI compliant or similar, then a proof of Reviews for all changes is at least a part of what is needed)
Also most of our Maintainers are volunteers and have limited time anyway. (and if we dont get to outsource at least a part of the Reviews to contributors, we would need to test more manually and need to decide alone about usecases we did not even use yet)

We have similar complaints of things taking to long, and the numbers show, that we are still getting more PRs in, then we are able to process. But also, there are quite some cases which are hard to test or have some discussions pending. Slowly is sometimes better to get more informed feedback <.<
https://cauldron.io/project/4125?from_date=2010-07-09&to_date=2021-07-09&tab=activity-overview
